Tests/UITests/TestApp/HealthKitTestsView/Sleep/SleepSessionsView.swift

```diff
@@ -17,10 +19,14 @@ struct SleepSessionsView: View {
     @Environment(HealthKit.self)
     private var healthKit
     
+    @SleepPhaseColors private var sleepPhaseColors
+    
     @HealthKitQuery(.sleepAnalysis, timeRange: .init({ () -> Range<Date> in
         let cal = Calendar.current
-        let start = cal.date(from: .init(year: 2025, month: 2, day: 19))! // swiftlint:disable:this force_unwrapping
-        let end = start.addingTimeInterval(60 * 60 * 24 * 2) // add 2 days, so that we fetch the entirety of the 19th and 20th
+//        let start = cal.date(from: .init(year: 2025, month: 2, day: 19))! // swiftlint:disable:this force_unwrapping
```

Comment:
Remove some commented out code

Reply:
thanks for catching this; actually needs to get reverted back to the old (commented-out) version